src/Components/CreateImageWizardV2/steps/Review/Footer/CreateDropdown.tsx

```diff
@@ -41,7 +43,19 @@ export const CreateSaveAndBuildBtn = ({
       (await createBlueprint({
         createBlueprintRequest: requestBody,
       }).unwrap()); // unwrap - access the success payload immediately after a mutation
-    blueprint && buildBlueprint({ id: blueprint.id, body: {} });
+
+    if (blueprint) {
+      buildBlueprint({ id: blueprint.id, body: {} });
+      dispatch(setBlueprintId(blueprint.id));
+
+      setTimeout(() => {
+        const row = document.getElementById(`blueprint-row-${blueprint.id}`);
+        if (row) {
+          row.click();
```

Comment:
I wonder why do we need to click?
In my mind we should: invalidate the Blueprints data so we refetch and then dispatch the blueprint ID to select the blueprint, right?

Or is the click needed for some other reasons?